app/src/main/res/layout/fragment_button_outline.xml

```diff
@@ -12,7 +12,7 @@
     android:id="@+id/btn_default"
     android:layout_width="wrap_content"
     android:layout_height="wrap_content"
-    android:text="Default"
+    android:text="button"
```

Comment:
minor, Upper case `B` in `button`

Reply:
Yes, version is wrong from the previous release